app/src/test/java/org/glucosio/android/fragment/AssistantFragmentTest.java

```diff
@@ -3,10 +3,13 @@
 import org.glucosio.android.RobolectricTest;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
 import org.robolectric.shadows.support.v4.SupportFragmentTestUtil;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@RunWith(RobolectricTestRunner.class)
```

Comment:
This is not needed! Check the `RobolectricTest` class

Reply:
Omg, missed that it extends RobolectricTest.class